modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -225,6 +225,11 @@ TEST_P(Test_ONNX_layers, Gather)
     testONNXModels("gather", npy, 0, 0, false, false);
 }
 
+TEST_P(Test_ONNX_layers, GatherElements)
+{
+    testONNXModels("gather_elements", pb);
+}
```

Comment:
Please do not forget to open a pull request to https://github.com/opencv/opencv_extra/ with the same branch name: `GSoC_Support_GatherElements_ONNX`

Reply:
done: https://github.com/opencv/opencv_extra/pull/1082